infrastructure/main.tf

```diff
@@ -25,7 +25,8 @@ data "azurerm_key_vault" "cmc_key_vault" {
 
 data "azurerm_key_vault_secret" "notify_api_key" {
   name = "notify-api-key"
-  vault_uri = "${data.azurerm_key_vault.cmc_key_vault.vault_uri}"
+  //vault_uri = "${data.azurerm_key_vault.cmc_key_vault.vault_uri}"
```

Comment:
do we need this ` vault_uri ` any more? if not we can remove this commented line

Reply:
We leave as it is for now. We will do infrastructure code cleanup seperately

Reply:
👍